src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
```

Comment:
It'd be good to add a brief description to this method and to the `purge()` method, and perhaps also a longer description to both mentioning the existence of the other; makes it easier for people using autocomplete. (Brief descriptions are one-liners, and detailed descriptions are separated below these by a blank line; see http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html for more.)

Also, I'd leave out the `@since`, because it's not clear yet what it'll be released as (although, @addshore do you have any guidelines about this? Are you using Semantic Versioning? If so, this should be `@since 0.7.0` as that's the next minor version number that we'll hit).